src/gui/addnewtorrentdialog.ui

```diff
@@ -393,7 +393,6 @@
  </customwidgets>
  <tabstops>
   <tabstop>savePath</tabstop>
-  <tabstop>defaultSavePathCheckBox</tabstop>
```

Comment:
This was probably done automatically.
If I am not mistaken this controls the tab order. So shouldn't you place the new widget's name here?

Reply:
Indeed. I noticed that the tab order in this file and in optiondlg is all messed up except for the RSS tab (probably doesn't have tabstops) and the advanced tab (no .ui). So i tried deleting all tabstops from optionsdlg.ui and the order was mostly restored except for some items which are added out of order.

The question is whether it is better to remove tabstops and fix the order of the items in the file or to fix the tabstops order?

Reply:
Just brought it back here. The tabstop order problem is out of this PR.

Reply:
>which are added out of order.

I don't think there are orders between widgets, the chaos is caused by adding spacers, once you add a spacer, the order of tabstop blows up... should be a bug in Qt.